state_manage/actions/actionTypes.ts

```diff
@@ -0,0 +1,2 @@
+export const SET_NAME = 'SET_NAME'
+export const SET_BIO = 'SET_BIO'
```

Comment:
New line at end of file

Reply:
Other screens will have additional reducers and actions. Just so that we don't get confused, can you do either of the following:
1. Rename your file into something like profileScreenActionTypes.ts, same for actions.ts and reducers.ts. Create a `reducers` folder to host the reducer files just like what you're doing for actions
2. Put comments before your code specifying that this is for profile screen.

I personally prefer 1.